src/main/java/com/indico/mutation/WorkflowSubmission.java

```diff
@@ -55,6 +56,17 @@ public WorkflowSubmission workflowId(int id) {
         return this;
     }
 
+    /**
+     * A UUID representing a unique set of files and workflow activity.
+     * This optional parameter helps the platform detect duplicate submissions.
```

Comment:
Maybe worth mentioning that it also prevents duplicate submissions? "helps the platform detect and prevent duplicate submissions." Maybe this is redundant though